main/precache-workload/main.go

```diff
@@ -91,20 +89,18 @@ func main() {
 
 	log.Info("Loaded precache spec file.")
 
-	// Change root directory to /host
-	if err := syscall.Chroot(common.Host); err != nil {
-		terminateOnError(fmt.Errorf("failed to chroot to %s, err: %w", common.Host, err), Failure)
-	}
-	log.Infof("chroot %s successful", common.Host)
-
```

Comment:
The pre-caching k8s job needs to chroot to the mounted host filesystem. May I ask why you removed this?

Reply:
I changed it to be the same as all other code. Now it uses chroot executor